### DISCUSSION TITLE
Shop Pay Addresses are not validated

### DISCUSSION THREAD

**Discussion by mufftee** [Shopify/function-examples#443] [Feedback: Cart and Checkout Validation API]
This is should be fixed asap. Thats leads to false addresses and also returning delivery for the merchant.

**Comment by briannaleung** [comment]
Hey! I just tested this in production. Seems to work as expected. Are you sure the correct configuration settings are applied in checkout settings? Checking that box for the checkout rule allows customers to checkout without the validations.

![image](https://github.com/Shopify/function-examples/assets/33970764/d219f19c-adb1-4c89-a644-9152778fdb37)

**Reply by mufftee** [reply]
@briannaleung all our functions are blocking on failure as it should be. We had a client report that Mastercards with Shop Pay wont be blocked. We have a function that checks if a number is included in address1, if not it blocks the checkout. Thats totally fine working for amazon, paypal, google, apple and also the shipping address. The only payment that wont get checked is Shop Pay, we could also reproduce that.

**Reply by mufftee** [reply]
Maybe it was an Outage at this exact moment that causes this behavior, but we tested it like 30 minutes later after the customer did report it. So that would be a long Outage :D

**Reply by mufftee** [reply]
Topic solved, we missed the CHECKOUT_COMPLETION BuyerYourney
